src/SitemapServiceProvider.php

```diff
@@ -27,6 +28,10 @@ public function boot()
             ->give(function () {
                 return Crawler::create(config('sitemap.guzzle_options'));
             });
+
+        $this->app->bind(Profile::class, function ($app, $params) {
+            return new Profile(reset($params));
```

Comment:
Why do you use `reset` here?

Reply:
I generally use it to return the first element of the array, but I can change to `$params[0]` if you want?

Reply:
Yeah, `params[0]` is more clear to me, I'll change this myself.